website/Bomberjam.Website/Frontend/scripts/react/viewerWithControls.tsx

```diff
@@ -67,6 +67,9 @@ export const ViewerWithControls = (props: VisualizerProps) => {
       replayCtrl?.destroy();
     }
   }, [props.loadingText]);
+  
+  const currentTick = props.gameHistory?.ticks[selectedStateIdx].state.tick;
+  const maxTicks = props.gameHistory?.ticks.length;
```

Comment:
It would also be possible to do ``selectedStateIdx + 1`` and ``maxStateIdx + 1``, but I feel this is clearer and more robust.